server/api/users.js

```diff
@@ -225,17 +239,13 @@ router.post('/users/new', apiLimiter, multer.single('avatarFile'), async (req, r
       await dbRef.doc(referrer).collection('referrals').doc(user).create(timestampObj);
     }
 
-    switch (platform) {
-      case 'facebook': {
-        const doc = {
-          [platform]: {
-            userId: platformUserId,
-          },
-        };
-        await oAuthDbRef.doc(user).create(doc);
-        break;
-      }
-      default:
+    if (platformUserId) {
+      const doc = {
+        [platform]: {
+          userId: platformUserId,
+        },
+      };
```

Comment:
probably want to add a comment that, this `platform` is guarded from user modification by the fact that only `platformUserId` is only set when the `platform` is valid.
or even better, have a function that filters valid platforms in this case? not sure